packages/appium/lib/cli/extension-command.js

```diff
@@ -639,9 +639,15 @@ class ExtensionCliCommand {
         const updateVer = unsafe && update.unsafeUpdate ? update.unsafeUpdate : update.safeUpdate;
         await spinWith(
           this.isJsonOutput,
-          `Updating driver '${e}' from ${update.current} to ${updateVer}`,
+          `Updating ${this.type} '${e}' from ${update.current} to ${updateVer}`,
           async () => await this.updateExtension(e, updateVer)
         );
+        if (!unsafe && update.unsafeUpdate) {
```

Comment:
I'am a bit lost there. There is quite similar condition above, that prints same message. How it is different from this one?

Reply:
The above condition is only printed when there is no safe update available. For example, if I'm on uiautomator2 driver 2.45.0 and I request an update, the driver will get updated to the latest 2.x, specifically, 2.45.1 (as expected), but with no information that 3.x even exists. This PR aims to cover exactly this case.

Reply:
could you please add it as an explanation comment to the source to avoid possible confusion?

Reply:
added